### DISCUSSION TITLE
How to show the progress of training process when using model.fit

### DISCUSSION THREAD

**Discussion by hieutv85** [timeseriesAI/tsai#780] [Q&A]
Hi there, 

I am a new user of tsai. I want to apply time series classification methods of tsai to multivariate time series data. 
My time series data is so large so when I apply model.fit it take a lot of time to run. 

Is there anyway to show the progress of the training process? (such as the verbose parameter of keras model.fit) 
Since it take a lot of time to run so I want to estimate how much time will it take. 

Thank you very much,

**Comment by hieutv85** [comment]
Hi guys, please help me.
Please support me @oguiza . Thank you very much.

Below is my code:

_from tsai.basics import *
tfms = [None, TSClassification()]
batch_tfms = TSStandardize(by_sample=True)
mv_clf = TSClassifier(X_sw,yr, splits=split, path='models', arch="InceptionTimePlus", tfms=tfms, batch_tfms=batch_tfms, metrics=accuracy, cbs=ShowGraph())
mv_clf.fit_one_cycle(10, 1e-2)_

**Reply by irazza** [reply]
How long the timeseries is ? 
The fit_one_cycle() method should print a "summary" of the training process for every epoch.

**Reply by hieutv85** [reply]
Hi @albertoazzari, 

The length of the time series is 1.5 million time lags. And I have 15 similar time series like that. 
I already trained one time series on a high performance computer and I cannot see the results after 3 days. 
If I train all of 15 time series and do not know about the progress then it will very inconvenient.

**Comment by oguiza** [comment]
Hi @hieutv85 , 
When training a model you get an updated with every batch. 
It doesn't make sense to train a model with a time series of 1.5  time steps. I don't know what you are trying to learn, but you should probably apply a sliding window.
